lib/src/js/utils.dart

```diff
@@ -33,6 +33,17 @@ external JSClass get jsErrorClass;
 /// Returns whether [value] is a JS Error object.
 bool isJSError(Object value) => instanceof(value, jsErrorClass);
 
+final _jsEquals = JSFunction('a', 'b', 'return a===b');
+
+/// Returns where [a] and [b] are considered equal in JS.
+bool jsEquals(Object? a, Object? b) => _jsEquals.call(a, b) == true;
+
+@JS("Symbol")
+class JSSymbol {}
+
+@JS("Symbol")
+external JSSymbol createJSSymbol();
```

Comment:
This is also unused now that we changed the way we initialize the importer.

Reply:
Removed in https://github.com/oddbird/dart-sass/commit/15e33b9b2fc0a9848076f2356a1c99680f8198bc